pkg/model/estimator/local/ratio_process.go

```diff
@@ -27,8 +27,51 @@ import (
 	"k8s.io/klog/v2"
 
 	collector_metric "github.com/sustainable-computing-io/kepler/pkg/collector/metric"
+	"github.com/sustainable-computing-io/kepler/pkg/config"
 )
 
+// getTransferredDataFromMetric is used when there is Hardware counter so we have to use cgroup or eBPF (CPUTime)
+// directly, currently for CoreUsageMetric we use cputime and for DRAM, use cgroup dram info
+func getTransferredDataFromMetric(containerMetrics *collector_metric.ContainerMetrics, usageMetrics string) float64 {
+	switch usageMetrics {
+	case config.CoreUsageMetric:
+		// the containerMetrics will be system container which contains all process metrics,
+		// now, we use the containerMetrics.CPUTime.Delta so we use the all CPUTime in this period
+		// as total CPUTime so we can divide by each process CPUTime
+		return float64(containerMetrics.CPUTime.Delta)
+	case config.DRAMUsageMetric:
+		// same to CPUTime, use Cgroup's memory value and sum all Delta value as summary thus
+		// we can divide from each process memory as well
+		return float64(containerMetrics.CgroupStatMap[config.CgroupfsMemory].SumAllDeltaValues())
```

Comment:
return float64(containerMetrics.CPUTime.Delta), right?

Reply:
no, for DRAM, I use the cgroup sum all metrics, not use CPUTime ..
I can certainly go back to CPUTime but want to know use CPUTime to represent DRAM is a good idea

Reply:
CPU time has better correlation to memory access than memory allocation (config.CgroupfsMemory)...

Reply:
Memory allocation alone does not provide insights into memory access. However, a busy CPU is more likely to be accessing memory. Although CPU utilization (CPUTime) does not directly represent memory access, it remains the only viable proxy available to approximate such information reliably.

Reply:
ok, will switch back to original implementation, thanks for the insight~